packages/app/config/webpack.common.js

```diff
@@ -55,6 +55,7 @@ module.exports = {
   entry: SANDBOX_ONLY
     ? {
         sandbox: [
+          '@babel/polyfill',
```

Comment:
I'm wondering if this would make the bundler bigger than it needs to be, since with this it's including all babel polyfills out there. Is there a way to make this more fine-grained? I thought that babel-preset-env would do something like this.

Reply:
Yeah, I think we only need a setImmediate poly... which can basically just be:

```
global.setImmediate = global.setImmediate || global.requestAnimationFrame
```

Will try to remove that and only use custom stuff inside existing polyfill file there